.github/workflows/labelsync.yml

```diff
@@ -0,0 +1,26 @@
+name: Automatic Label Sync
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+
+jobs:
+  label_sync:
+    name: Automatic Label Synchronization
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      issues: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+        with:
+          sparse-checkout: .github/labels.yml
+          sparse-checkout-cone-mode: false
+          repository: Compairifuel/.github
+      - name: Run Label Sync
+        uses: crazy-max/ghaction-github-labeler@de749cf181958193cb7debf1a9c5bb28922f3e1b # v5
```

Comment:
Is there a reason you are using a third party?

Reply:
As stated in our meeting: I think we should focus more on the API and app for now.
However, I have created [#8](https://github.com/Compairifuel/.github/issues/8) for when we want to get to it.